server/utils/calculateConfidence.ts

```diff
@@ -0,0 +1,17 @@
+import { ConfidenceValue } from "../models/room.model.ts";
+
+const calculateConfidence = (timeStarted: Date): ConfidenceValue => {
+	const currentTime = new Date();
+	const millisecondsElapsed = currentTime.getTime() - timeStarted.getTime();
+	const secondsElapsed = millisecondsElapsed / 1000;
+
+	if (secondsElapsed < 15) {
+		return ConfidenceValue.high;
+	}
+	if (secondsElapsed < 45) {
```

Comment:
I arbitrarily chose these numbers to start.  If we switch to having a countdown timer, then we can switch this up.

Reply:
Hmm, 15 seconds seems a little slow for high confidence. What about like 5 or even 3 seconds? Then maybe medium is 15 seconds or less?

Reply:
Sure.  I'll switch it to 5 and 15 for now.